tests/integration/targets/azure_ops_test_azure_manage_snapshot/runme.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
no runme.sh in other integration tests

Reply:
This is normal if you need to split the tests between different hosts.  The azure snapshot operations have to happen from localhost and to validate the snapshot operations have to happen on the VirtualMachine under test.

This is the accepted way to do this under ansible-test

Reply:
I see. Ok. I'll check tomorrow if the existing Jenkins job works with playbooks